scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -68,6 +68,6 @@ You also need, in your msys2 environment (adjust paths for your system):
   export BAZEL_SH=c:/tools/msys64/usr/bin/bash.exe
   export BAZEL_PYTHON=c:/tools/python2/python.exe
 
-See also https://bazel.build/docs/windows.html
+See also https://bazel.build/versions/master/docs/windows.html
```

Comment:
For now, we should use docs/windows.html since it is shorter and will automatically redirect to windows.html page of the latest version under versions/.

FYI Once I finish the script for cutting releases of the docs, ideally, this script should print the link with the correct version, such as https://bazel.build/versions/0.5/docs/windows.html. We can make that change separately once versioned docs is done.